config/analytics.yml

```diff
@@ -26,6 +26,14 @@ shared:
     - id
     - name
     - updated_at
+  authentication_tokens:
+    - id
+    - hashed_token
+    - enabled
+    - provider_id
+    - expires_at
+    - created_at
+    - updated_at
```

Comment:
Should we avoid sending these to DfE Analytics?

Reply:
these should go in `config/analytics_blocklist.yml`